### PR TITLE
chore: add volunteer sumanpaikdev

### PR DESCRIPTION
**Pull request by sumanpaikdev**
Followed instructions via Readme.md file.

### CODE REVIEW

File: data/users/sumanpaikdev.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Suman Paik",
+    "bio": "Frontend Web Developer",
+    "image": "https://avatars.githubusercontent.com/u/93247057?v=4",
+    "categories": [
+        {
+            "slug": "web-development",
+            "topics": ["html", "css", "javascript", "react", "java" ]
```
Comment (codewithvoid) [inline data/users/sumanpaikdev.json:8]: move javascript and java to programming category. like below
```json
[
  {
    "slug": "web-development",
    "topics": [
      "html",
      "css",
      "react"
    ]
  },
  {
    "slug": "programming",
    "topics": [
      "javascript",
      "java"
    ]
  }
]
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @sumanpaikdev for volunteering. Added a small comment.

**Review by sumanpaikdev — COMMENTED** [review]
updated .json

File: data/users/sumanpaikdev.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            ]
+        },
+        {
+            "slug": "programming",
+            "topics": [
+                "javascript",
+                "java"
+            ]
+        }
+    ]
```
Comment (codewithvoid) [inline data/users/sumanpaikdev.json:21]: add twitter contact here

**Review by sumanpaikdev — COMMENTED** [review]
Uhh! I really need some sleep.

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes. Welcome to the community :)

**Pull request merged**